tests/integration_tests/clouds.py

```diff
@@ -143,20 +144,48 @@ def _get_cloud_instance(self):
         )
 
 
-class LxdContainerCloud(IntegrationCloud):
-    datasource = 'lxd_container'
-    integration_instance_cls = IntegrationLxdContainerInstance
+class _LxdIntegrationCloud(IntegrationCloud):
+    integration_instance_cls = IntegrationLxdInstance
 
     def _get_cloud_instance(self):
-        return LXD(tag='lxd-integration-test')
+        return self.pycloudlib_instance_cls(tag=self.instance_tag)
```

Comment:
Just FYI, my type checking does complain about this because these variables can't be found in the current class, but since we don't have full type support in our python version, I'm not concerned about it. Looks like we'd need 3.6 to really fix it. https://www.python.org/dev/peps/pep-0526/